upcloud/load_balancer.go

```diff
@@ -207,6 +218,10 @@ type LoadBalancerBackendProperties struct {
 	HealthCheckExpectedStatus int                              `json:"health_check_expected_status,omitempty"`
 	StickySessionCookieName   string                           `json:"sticky_session_cookie_name,omitempty"`
 	OutboundProxyProtocol     LoadBalancerProxyProtocolVersion `json:"outbound_proxy_protocol,omitempty"`
+	TLSEnabled                *bool                            `json:"tls_enabled,omitempty"`
+	TLSVerify                 *bool                            `json:"tls_verify,omitempty"`
+	TLSUseSystemCA            *bool                            `json:"tls_use_system_ca,omitempty"`
+	HTTP2Enabled              *bool                            `json:"http2_enabled,omitempty"`
```

Comment:
Do we need pointers when default values are `false` ?

Reply:
I suppose we don't. `HealthCheckTLSVerify` could be changed to `bool` as well then, as it also defaults to `false`. But that would be a breaking change. Is changing that worth it, what do you think?

Reply:
As discussed, makes request part problematic so lets keep them as you have proposed :+1: